tests/instancetype/instancetype.go

```diff
@@ -596,20 +595,17 @@ var _ = Describe("[crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-c
 			newVM := libvmi.NewVirtualMachine(newVMI,
 				libvmi.WithInstancetype(instancetype.Name),
 				libvmi.WithPreference(preference.Name),
+				libvmi.WithRunStrategy(virtv1.RunStrategyAlways),
 			)
 			newVM, err = virtClient.VirtualMachine(testsuite.GetTestNamespace(vm)).Create(context.Background(), newVM, metav1.CreateOptions{})
 			Expect(err).ToNot(HaveOccurred())
-
-			By("Waiting for a VirtualMachineInstancetypeSpec ControllerRevision to be referenced from the new VirtualMachine")
-			Eventually(func() string {
-				newVM, err = virtClient.VirtualMachine(newVM.Namespace).Get(context.Background(), newVM.Name, metav1.GetOptions{})
-				if err != nil {
-					return ""
-				}
-				return newVM.Spec.Instancetype.RevisionName
-			}, 300*time.Second, 1*time.Second).ShouldNot(BeEmpty())
```

Comment:
Slightly unrelated but as  above, we should use some shared code for these checks otherwise things like this happen with different authors using different approaches.

Reply:
This is out of the scope of this PR